docs/MO_DG/prepare_model/FP16_Compression.md

```diff
@@ -1,10 +1,12 @@
 # Compressing a Model to FP16 {#openvino_docs_MO_DG_FP16_Compression}
 
```

Comment:
I think now we should have documentation for generation IR in the original precision when we want to disable FP16 compression. So the current topic is no longer relevant.

Reply:
Updated this doc. Please review